examples/send-zerocopy.c

```diff
@@ -62,16 +62,16 @@ static char payload[IP_MAXPACKET] __attribute__((aligned(4096)));
  */
 void t_error(int status, int errnum, const char *format, ...)
 {
-    va_list args;
-    va_start(args, format);
+    	va_list args;
+    	va_start(args, format);
 
-    vfprintf(stderr, format, args);
-    if (errnum)
-	fprintf(stderr, ": %s", strerror(errnum));
+    	vfprintf(stderr, format, args);
+    	if (errnum)
+		fprintf(stderr, ": %s", strerror(errnum));
 
-    fprintf(stderr, "\n");
-    va_end(args);
-    exit(status);
+    	fprintf(stderr, "\n");
+    	va_end(args);
+    	exit(status);
```

Comment:
> Use tabs instead of spaces.
> Signed-off-by: Steffen Winter <steffen.winter@proton.me>

No way, don't add a fixup to fix the mistake in the same series.
Please squash this.

Reply:
My bad